tests/snapshot_in_svnn_transition_test.py

```diff
@@ -101,20 +104,31 @@ def takeSnapshot(node):
     info = cluster.biosNode.getInfo(exitOnError=True)
     assert (info["head_block_num"] - info["last_irreversible_block_num"]) < 9, "Instant finality enabled LIB diff should be small"
 
-    def restartWithSnapshot(node):
+    def restartWithSnapshot(node, rmBlocks, addSnapshot=True):
        Print("Shut down node")
        node.kill(signal.SIGTERM)
        Print("Restart node with snapshot")
-       node.removeState()
+       Utils.rmNodeDataDir(node.nodeId, rmBlocks=rmBlocks)
```

Comment:
Should `rmNodeDataDir()` belong to `node` class? If too many places to change, then  leave as is.